fortanix-sgx-tools/src/bin/ftxsgx-runner-cargo.rs

```diff
@@ -98,6 +98,10 @@ fn run() -> Result<(), Error> {
 
     let mut ftxsgx_runner_command = Command::new(runner);
     ftxsgx_runner_command.arg(args[1].clone() + ".sgxs");
+    if args.len() > 2 {
+        ftxsgx_runner_command.arg("--");
+        ftxsgx_runner_command.args(&args[2..]);
```

Comment:
What happens if someone runs `runner <enclave> --`?

Reply:
Then `&args[2..]` returns the empty slice so I think this is fine

Reply:
Also when run with cargo, as is the intention, what does this look like?
`cargo run -- -- <enclave arg>`? IE do we need one `--` to get arguments to the runner, and another `--` to get arguments to the enclave?

Reply:
Just one `--` would be enough in that case. That's actually why I added `--` in the code here so you don't have to repeat it twice.